en/docs/developer-guides/tools/migration-tools/logic-apps-migration-tool.md

```diff
@@ -0,0 +1,463 @@
+# Azure Logic Apps migration tool
```

Comment:
1. Page title and file name should match as file name is what is appearing in URL.
2. Make the L1 title , Title case.

Reply:
Addressed review suggestions